src/background/background.js

```
@@ -48,106 +420,426 @@ chrome.commands.onCommand.addListener((command) => {
+    
+    // 1. Save the full HTML content
+    const htmlFilename = `${filename}.html`;
+    const htmlContent = snapshot.htmlContent || '<html><body><h1>HTML content not available</h1></html>';
+    downloadTextFile(htmlContent, htmlFilename);
+    
+    // 2. Save the structured metadata as JSON
+    // Remove the full HTML content from the JSON to keep file size reasonable
+    const jsonSnapshot = { ...snapshot };
+    delete jsonSnapshot.htmlContent;
+    
+    const jsonFilename = `${filename}.json`;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)